### DISCUSSION TITLE
Ender 3 S1 with TJC Screen

### DISCUSSION THREAD

**Discussion by elberto22** [mriscoc/Ender3V2S1#1313] [Q&A]
Hey! i have an ender 3 s1 with an F4 chip and a TJC color screen, however whenever i try to flash the firmware on my printer the screen is stuck on the creality logo, however when i play around with the knob i do hear beeps and can make the printer move, so to my knowledge is that the firmware flashes but my screen is blank. any ideas why?

**Comment by mriscoc** [comment]
Check here: https://github.com/mriscoc/Ender3V2S1/issues/966

**Reply by elberto22** [reply]
I just saw that like 20 minutes ago and turns out i need a smaller micro sd card, ordered one on amazon and ill try that, thanks!

**Reply by elberto22** [reply]
the new sd cards worked flawlessly! this is a really cool thing you've made!
